tests/e2e/tests/content-manager/listview.spec.ts

```diff
@@ -58,10 +58,43 @@ test.describe('List View', () => {
       const publishButton = page.getByRole('button', { name: 'Publish' }).first();
       await publishButton.click();
 
-      // Wait for the selected entries modal to appear
-      await page.waitForSelector(
-        'text=0 entries already published. 2 entries ready to publish. 0 entries waiting for action'
-      );
+      await page.waitForSelector('text=Ready to publish');
+
+      const verifyPublicationStatus = async (
+        publishedCount: number,
+        readyToPublishCount: number,
+        waitingForActionCount: number,
+        readyToPublishChangesCount: number
+      ) => {
+        const alreadyPublishedElement = await page
+          .locator('text=Already Published')
```

Comment:
We should try to use the recommended built in locators: https://playwright.dev/docs/locators#quick-guide. Have you tried getByText?